pennylane/transforms/qcut.py

```diff
@@ -1709,6 +1712,12 @@ def qcut_processing_fn(
         float or tensor_like: the output of the original uncut circuit arising from contracting
         the tensor network of circuit fragments
     """
+    if qml.active_return():
+        results = [
+            qml.math.stack(r) if isinstance(r, tuple) else qml.math.reshape(r, [-1])
```

Comment:
Similar comment as above.

Reply:
See my other comment. Here we also need to expand the dimensions if there is a single measurement, since there's a concatenation after this line. This is because in the old return types there is already an extra leading dimension for a single measurement.